sdk-extensions/tracing-incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/trace/WorkerExporter.java

```diff
@@ -111,7 +111,10 @@ public void flush(Collection<SpanData> batch, AbstractQueue<ReadableSpan> queue)
       }
     }
     exportCurrentBatch(batch);
-    flushSignal.get().succeed();
-    flushSignal.set(null);
+    CompletableResultCode flushResult = flushSignal.get();
```

Comment:
Is this actually possible?

Reply:
Doubt it - I could use a requireNonNull instead to signal that but I guess this is still a bit more robust to refactoring.

Reply:
Please add a comment then that we do not expect it to be null. Otherwise readers of the code may be left wondering. (also a warning log might make sense, but I guess that's a difficult topic)